docs/development.rst

```diff
@@ -270,128 +270,39 @@ Normally you can install the client the same way on Windows as on Linux, like so
-
-* `Anaconda <https://anaconda.org/>`_
-* `Miniforge on GitHub <https://github.com/conda-forge/miniforge>`_
-* `Conda-forge <https://conda-forge.org/>`_
-
-The instructions below should work in both Anaconda and Miniforge.
-Though with Miniforge you can simplify the commands a little bit because the conda-forge channel is the default, so you can leave out the option ``-c conda-forge``.
-
-Create a conda environment with the geopandas package already installed.
-Installing geopandas from a conda package is the step that avoids the hard part.
+1. Create and activate a new conda environment for developing the openeo-python-client.
+2. In that conda environment, install only the dependencies of openeo via conda, but not the openeo package itself.
+3. Do a pip install of the code in *editable mode* with `pip -e`.
 
 .. code-block:: console
 
-    conda create -n <your environment's name>  geopandas
-
-    # for example
-    conda create -n openeopyclient  geopandas
+    conda create -n <your environment's name>
 
-Activate the conda environment
-
-.. code-block:: console
+    # For example:
+    conda create -n openeopyclient
 
+    # Activate the conda environment
     conda activate openeopyclient
 
-Next, run the dev install with pip
-
-In the directory where you git-cloned the openEO Python client:
-
-.. code-block:: console
-
-    python -m pip install -e .[dev]
-
-A quick way to check whether the client was successfully installed or not is to print its version number.
-
-In your conda environment, launch the Python interpreter and try the following snippet of Python code to show the client's version:
+    # Install openeo from the conda-forge channel
+    conda install --only-deps -c conda-forge openeo
 
-.. code-block:: python
-
-    import openeo
-
-    print(openeo.client_version())
-
-
-.. _windows-dev-install-unofficial-wheels:
-
-Option 2) Use some unofficial python wheels for GDAL and Fiona
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-There are `unofficial Python wheels at https://www.lfd.uci.edu/~gohlke/pythonlibs/: <https://www.lfd.uci.edu/~gohlke/pythonlibs/>`_
-
-But as the name says, these wheels have no official support, so they are not recommended for production.
-They can however help you out for a development environment.
+    # Now install the openeo code in editable mode.
+    pip install -e .[dev]
 
-You need to install the wheels for GDAL and Fiona.
 
-* wheels for `Fiona <https://www.lfd.uci.edu/~gohlke/pythonlibs#fiona>`_
-* wheels for `GDAL <https://www.lfd.uci.edu/~gohlke/pythonlibs/#gdal>`_
+There is a also shorter version of the above, if you prefer.
+You can create the environment and install openeo's dependencies in one go:
 
-.. code-block::
+.. code-block:: console
 
-    # In your activate virtualenv
-    # install the wheels:
-    pip install <path to GDAL whl file> <path to fiona whl file>
+    conda create -n openeopyclient --only-deps -c conda-forge openeo
```

Comment:
I would not provide this short version in the docs, doing `conda create` with `--only-deps` might get ugly if users also add other packages to the `conda create`  line

Reply:
Good point. I removed that paragraph.